deltachat-jsonrpc/src/api.rs

```diff
@@ -896,19 +896,37 @@ impl CommandApi {
             .to_u32())
     }
 
-    // for now only text messages, because we only used text messages in desktop thusfar
+    /// Add a message to the device-chat.
+    /// Device-messages usually contain update information
+    /// and some hints that are added during the program runs, multi-device etc.
+    /// The device-message may be defined by a label;
+    /// if a message with the same label was added or skipped before,
+    /// the message is not added again, even if the message was deleted in between.
+    /// If needed, the device-chat is created before.
+    ///
+    /// Sends the event MsgsChanged event on success.
+    ///
+    /// Setting msg to None will prevent the device message with this label from being added in the future.
```

Comment:
Just out of curiousity: For what is this feature (blocking labels) required?

Reply:
(code looks fine as far as I can tell :+1:)

Reply:
> Just out of curiousity: For what is this feature (blocking labels) required?

https://github.com/deltachat/deltachat-desktop/issues/3637